docs/api/api_blueprint.apib

```diff
@@ -301,7 +301,7 @@ Authorized user should be same as user in request body or must be admin.
                 "logo-width": null,
                 "logo-height": null
               },
-              "type": "image-size",
+              "type": "image_size",
```

Comment:
These types should also be dasherized for the jsonapi clients to understand what to use.